src/www/views/servers_view/server_connection_indicator/index.ts

```diff
@@ -180,3 +185,5 @@ export class ServerConnectionIndicator extends LitElement {
     ].includes(state);
   }
 }
+
+customElements.define('server-connection-indicator', ServerConnectionIndicator);
```

Comment:
Alright so, it seems the post-processor we must use for lit ([`postcss-lit`](https://www.npmjs.com/package/postcss-lit)) silently fails when we use the decorator syntax, so I've moved this back to the older `customElements` to confirm that everything's working otherwise.

Unfortunately there's no public forum for `postcss-lit`, but doing some detective work it seems it's based on [`@stylelint/postcss-css-in-js`](https://github.com/stylelint/postcss-css-in-js) which is [soon to be deprecated](https://github.com/stylelint/postcss-css-in-js/issues/225).

In _that_ repository, there are a string of issues about babel errors getting suppressed: https://github.com/stylelint/postcss-css-in-js/issues/247. They claim to have fixed this in the current version, but afaict both latest `postcss-lit` and `@stylelint/postcss-css-in-js` fail in the same way.

We can't have compilation errors silently failing. **This will need to be solved before we can move to lit, since we also use postcss for doing the RTL mangling.** I have to look through the source and see if they read our babel config anywhere, or if there's anything else we can do. This may very well be an issue in our current Lit components in the Manager.

Reply:
Note, it seems that we're currently only rtl'ing for Polymer: https://github.com/Jigsaw-Code/outline-client/blob/41d45ad6823edd86cd454f1683987e244cd162ab/scripts/rtl_css_webpack.js#L30-L42

Reply:
I don't understand. What part of the compilation is failing?
Isn't the problem with the linter?

Reply:
The linter is compiling the files it finds. The compilation depends on `postcss-lit`. For us to use postcss with lit, we may have to use `postcss-lit`, as well. Currently we only use postcss with Polymer.

Reply:
Found the source for postcss-lit! https://github.com/Jigsaw-Code/outline-client/pull/1311#discussion_r871709722